drivers/sensor/lis2dw12/Kconfig

```diff
@@ -0,0 +1,271 @@
+
+# Copyright (c) 2017 STMicroelectronics
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig LIS2DW12
+	bool "LIS2DW12 I2C/SPI Three Axis Accelerometer"
+	depends on SENSOR && (I2C || SPI)
+	default n
+	help
+	  Enable driver for LIS2DW12 I2C-based acc sensor.
+
+config LIS2DW12_NAME
+	string "Driver name"
+	default "LIS2DW12"
+	depends on LIS2DW12
+	help
+	  Device name with which the LIS2DW12 sensor is identified.
+
+choice
+	prompt "Interface to AP"
+	depends on LIS2DW12
```

Comment:
Instead of repeating `depends on LIS2DW12` throughout, wrap everything in a big `if LIS2DW12..endif` instead.